src/axom/quest/examples/quest_shape_in_memory.cpp

```diff
@@ -0,0 +1,1410 @@
+  tetMesh.appendNode(+lll, -lll, +lll);
+  axom::IndexType conn0[4] = {0, 1, 2, 3};
+  tetMesh.appendCell(conn0);
+  axom::IndexType conn1[4] = {4, 5, 6, 7};
+  tetMesh.appendCell(conn1);
+
+  SLIC_ASSERT(axom::mint::blueprint::isValidRootGroup(meshGroup));
+
+  axom::klee::TransformableGeometryProperties prop {
+    axom::klee::Dimensions::Three,
+    axom::klee::LengthUnit::unspecified};
+
+  auto compositeOp = std::make_shared<axom::klee::CompositeOperator>(startProp);
+  addScaleOperator(*compositeOp);
+  addRotateOperator(*compositeOp);
+  addTranslateOperator(*compositeOp, -1, 1, 1);
+
+  axom::klee::Geometry tetMeshGeometry(prop,
+                                       tetMesh.getSidreGroup(),
+                                       topo,
+                                       compositeOp);
+  axom::klee::Shape tetShape("tetmesh", "TETMESH", {}, {}, tetMeshGeometry);
+
+  return tetShape;
+}
+
+axom::klee::Geometry createGeometry_Vor(
+  axom::primal::Point<double, 3>& vorBase,
+  axom::primal::Vector<double, 3>& vorDirection,
+  axom::Array<double, 2>& discreteFunction,
+  std::shared_ptr<axom::klee::CompositeOperator>& compositeOp)
+{
+  axom::klee::TransformableGeometryProperties prop {
+    axom::klee::Dimensions::Three,
+    axom::klee::LengthUnit::unspecified};
+
+  SLIC_ASSERT(params.scaleFactors.empty() || params.scaleFactors.size() == 3);
+
+  const axom::IndexType levelOfRefinement = params.refinementLevel;
+  axom::klee::Geometry vorGeometry(prop,
+                                   discreteFunction,
+                                   vorBase,
+                                   vorDirection,
+                                   levelOfRefinement,
+                                   compositeOp);
+  return vorGeometry;
+}
+
+axom::klee::Shape createShape_Vor()
+{
+  Point3D vorBase = params.center.empty() ? Point3D {0.0, 0.0, 0.0}
+                                          : Point3D {params.center.data()};
+  axom::primal::Vector<double, 3> vorDirection = params.direction.empty()
+    ? primal::Vector3D {0.1, 0.2, 0.4}
+    : primal::Vector3D {params.direction.data()};
+  const int numIntervals = 5;
+  axom::Array<double, 2> discreteFunction({numIntervals + 1, 2},
+                                          axom::ArrayStrideOrder::ROW);
+  double zLen = params.length < 0 ? 1.6 : params.length;
+  double zShift = -zLen / 2;
+  double maxR = params.radius < 0 ? 0.75 : params.radius;
+  double dz = zLen / numIntervals;
+  discreteFunction[0][0] = 0 * dz + zShift;
+  discreteFunction[0][1] = 0.0 * maxR;
+  discreteFunction[1][0] = 1 * dz + zShift;
+  discreteFunction[1][1] = 0.8 * maxR;
+  discreteFunction[2][0] = 2 * dz + zShift;
+  discreteFunction[2][1] = 0.4 * maxR;
+  discreteFunction[3][0] = 3 * dz + zShift;
+  discreteFunction[3][1] = 0.5 * maxR;
+  discreteFunction[4][0] = 4 * dz + zShift;
+  discreteFunction[4][1] = 1.0 * maxR;
+  discreteFunction[5][0] = 5 * dz + zShift;
+  discreteFunction[5][1] = 0.0;
```

Comment:
Please add a comment about what `discreteFunction` is setting up

Reply:
Done.